src/jump.jl

```diff
@@ -8,18 +8,143 @@ Note that if non-linear constraint are present in the model, they are ignored.
 """
 hrep(model::JuMP.Model) = LPHRepresentation(model)
 
+function _get_all_constraint_ref_array(model::Model)
+    ref_arr = vcat([
+        all_constraints(model, c...)
+        for c in list_of_constraint_types(model)
+    ]...)
+end
+
+function _get_all_constraint_array(model::Model)
+    constraint_arr = constraint_object.(
+        _get_all_constraint_ref_array(model)
+    )
+end
+
+function _get_var_bounds_array(model::Model)
+    constraint_arr = _get_all_constraint_array(model)
+    var_bounds = filter(c -> c.func isa VariableRef, constraint_arr)
```

Comment:
You will be missing `VectorOfVariables`-in-`Zeros/Nonpositives/Nonnegatives`.

Reply:
Updated.